genrl/deep/agents/ddpg/ddpg.py

```diff
@@ -128,8 +128,9 @@ def __init__(
         self.create_model()
 
     def create_model(self):
-        state_dim = self.env.observation_space.shape[0]
-        action_dim = self.env.action_space.shape[0]
+        state_dim, action_dim, discrete = self.get_env_properties()
+        if discrete == True:
+            raise NotImplementedError
```

Comment:
Can you add a message to this error?

Reply:
Ok, I am changing the message to "Discrete environments not supported".

Reply:
"...not supported for {}".format(algo_name)

Reply:
oh sorry okay.